Casks/livereload.rb

```diff
@@ -3,9 +3,13 @@
   sha256 "1c915d4956177cfa5981896cb060fd00743f3d74157ad4e189901d0123fd8c7f"
 
   url "http://download.livereload.com/LiveReload-#{version}.zip"
-  appcast "https://s3.amazonaws.com/download.livereload.com/LiveReload-Mac-appcast.xml"
   name "LiveReload"
   homepage "http://livereload.com/"
 
+  livecheck do
+    url "http://livereload.com/mac/"
+    regex(/LiveReload-(\d+(?:\.\d+)*)\.zip/i)
```

Comment:
```suggestion
    regex(/href=.*?LiveReload[._-]v?(\d+(?:\.\d+)+)\.zip/i)
```

For what it's worth, the URL on this page is `https://s3.amazonaws.com/download.livereload.com/LiveReload-2.3.84.zip` but the cask URL would be `http://download.livereload.com/LiveReload-2.3.84.zip`. Should the cask `url` be updated to use the S3 URL?

Reply:
I'd say it is preferable to have a URL which matches the homepage domain.